app/api/jobs/channels/route.ts

```diff
@@ -1,35 +1,49 @@
+import processChannelReward from "@/lib/sonata/channelWeeklyAirdrop/processChannelReward";
+import processStakersReward from "@/lib/sonata/channelWeeklyAirdrop/processStakersReward";
 import sortChannels from "@/lib/sortChannels";
 import { stack } from "@/lib/stack/client";
+import { eventTipChannel } from "@/lib/stack/events";
 import getChannelStats from "@/lib/supabase/getChannelStats";
 import { NextRequest } from "next/server";
 
 export async function GET(req: NextRequest) {
   const TOP_CHANNELS = 10;
-  const TIP_AMOUNT = 11111;
+  const AIRDROP_AMOUNT = 5555;
```

Comment:
why is `AIRDROP_AMOUNT` changing?
- each airdrop should still be the same amount. we are simply dividing that amount in new ways.

Reply:
`AIRDROP_AMOUNT` is now  `11111/2` but rounded down due to unsupported floating numbers on the stack.